requirements-py2.txt

```diff
@@ -0,0 +1,12 @@
+#
+# These requirements were autogenerated by pipenv
+# To regenerate from the project's Pipfile, run:
+#
+#    pipenv lock --requirements
+#
+
+-i https://pypi.org/simple
+aenum==3.1.0; python_version < '3.6'
+future==0.18.2
+pycountry==18.12.8
+pyproj==2.2.2
```

Comment:
The toplevel requirements file for the data.govt.nz core repo had recorded the previously installed/frozen versions